railties/test/isolation/abstract_unit.rb

```diff
@@ -622,7 +622,7 @@ def self.sh(cmd)
   # Fake 'Bundler.require' -- we run using the repo's Gemfile, not an
   # app-specific one: we don't want to require every gem that lists.
   contents = File.read("#{app_template_path}/config/application.rb")
-  contents.sub!(/^Bundler\.require.*/, "%w(importmap-rails).each { |r| require r }")
+  contents.sub!(/^Bundler\.require.*/, "%w(sprockets/railtie importmap-rails).each { |r| require r }")
```

Comment:
Do we need to do this? `sprockets-rails` already require this file, so `Bundler.require` should just work.